codec-http/src/main/java/io/netty/handler/codec/http/websocketx/WebSocket08FrameDecoder.java

```diff
@@ -396,23 +398,34 @@ private void unmask(ByteBuf frame) {
 
         // Remark: & 0xFF is necessary because Java will do signed expansion from
         // byte to int which we don't want.
-        int intMask = ((maskingKey[0] & 0xFF) << 24)
-                    | ((maskingKey[1] & 0xFF) << 16)
-                    | ((maskingKey[2] & 0xFF) << 8)
-                    | (maskingKey[3] & 0xFF);
+        long longMask = ((long) maskingKey[0] & 0xff) << 56 |
```

Comment:
why reading from the same positions twice?
I suggest to read in local variables (probably the JIT will do it anyway, but better be safe then sorry) and
you can decide, if afailable, to make use of `PlatformDependent::getByte` too (and see how it compares with your version)

Reply:
I check with 4 local variables and PlatformDependent::getByte and not see any improvements in benchmarks. from the same position.